pkgs/dkg/drand.go

```diff
@@ -203,23 +203,10 @@ func (o *LocalOwner) PostDKG(res *kyber_dkg.OptionResult) error {
 	}
 	// Sign.
 	depositPartialSignature := secretKeyBLS.SignByte(signingRoot[:])
-	if depositPartialSignature == nil {
-		return fmt.Errorf("failed to sign deposit data with partial signature %w", err)
-	}
-	// Validate partial signature
-	if val := depositPartialSignature.VerifyByte(secretKeyBLS.GetPublicKey(), signingRoot[:]); !val {
-		err = fmt.Errorf("partial deposit root signature is not valid %x", depositPartialSignature.Serialize())
-		return err
-	}
 	// Sign SSV owner + nonce
 	data := []byte(fmt.Sprintf("%s:%d", eth_common.Address(o.data.init.Owner).String(), o.data.init.Nonce))
 	hash := eth_crypto.Keccak256([]byte(data))
 	sigOwnerNonce := secretKeyBLS.SignByte(hash)
-	// Verify partial SSV owner + nonce signature
-	val := sigOwnerNonce.VerifyByte(secretKeyBLS.GetPublicKey(), hash)
-	if !val {
-		return fmt.Errorf("partial owner + nonce signature isnt valid %x", sigOwnerNonce.Serialize())
-	}
```

Comment:
@GalRogozinski  suggested that its overkill to check sig after we created one, agree here, maybe its too much.

Reply:
wdym after we've created one? we're taking this from file input no?

if so then it might have been created by someone else or manipulated since creation